packages/components/app/styles/components/side-nav.scss

```diff
@@ -0,0 +1,243 @@
+  padding: var(--token-side-nav-wrapper-padding-vertical) var(--token-side-nav-wrapper-padding-horizontal);
+  overflow-y: auto;
+}
+
+.hds-side-nav__wrapper-footer {
+  padding: var(--token-side-nav-wrapper-padding-vertical) var(--token-side-nav-wrapper-padding-horizontal);
+}
+
+// * Header child elements
+// ---------------------------------------
+
+.hds-side-nav-header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.hds-side-nav-header__logo {
+  display: block;
+  align-items: center;
+  justify-content: center;
+  width: var(--token-side-nav-header-home-link-logo-width);
+  height: var(--token-side-nav-header-home-link-logo-height);
+}
+
+.hds-side-nav-header__actions {
+  display: flex;
+
+  > * + * { margin-left: var(--token-side-nav-header-actions-spacing); }
+}
+
+.hds-side-nav__home-link {
+  display: block;
+  width: 100%;
+  height: 100%;
+  padding: var(--token-side-nav-header-home-link-padding); // by design
+  // TODO discuss with Cristiano/Christian if this needs to become a component-level token
+  color: var(--token-color-foreground-high-contrast);
+  border: 1px solid transparent;
+  border-radius: 5px;
+
+  &:focus,
+  &.mock-focus {
+    @include hds-focus-ring-with-pseudo-element();
+  }
+
+  &:hover,
+  &.mock-hover {
+    background: var(--token-side-nav-interactive-surface-hover);
+  }
+
+  &:active,
+  &.mock-active {
+    background: var(--token-side-nav-interactive-surface-active);
+  }
+}
+
+// class name only, no component:
+.hds-side-nav__dropdown {
+  .hds-dropdown-toggle-icon {
+    color: var(--token-side-nav-foreground-primary);
+
+    &:hover {
+      background: var(--token-side-nav-interactive-surface-hover);
+    }
+
+    &:active {
+      background: var(--token-side-nav-interactive-surface-active);
+    }
+  }
+}
+
+// * Body child elements
+// ---------------------------------------
+.hds-side-nav__body {
+  flex: 1;
+  overflow-y: auto;
+}
+
+.hds-side-nav__list-wrapper {
+  & + & { margin-top: var(--token-side-nav-body-list-margin-vertical); }
+}
+
+.hds-side-nav__list-title {
+  min-height: 34px;
+  padding: 0 var(--token-side-nav-body-list-item-padding-horizontal);
+  color: var(--token-side-nav-foreground-faint);
+}
+
+.hds-side-nav__list {
+  margin: 0;
+  padding: 0;
+
+  // TODO: this doesn't work because Ember adds whitespace inside the ul
+  &:empty { display: none; }
+}
+
+.hds-side-nav__list-item {
+  display: flex;
+  align-items: center;
+  list-style-type: none; // Q: Should ul have role="list" added for accessibility?
+
+  &:last-child { margin-bottom: 0; }
+}
+
+.hds-side-nav__list-item-link {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  min-height: 36px;
+  padding: var(--token-side-nav-body-list-item-padding-vertical) var(--token-side-nav-body-list-item-padding-horizontal);
+  color: var(--token-side-nav-foreground-primary);
+  text-decoration: none;
+  background: var(--token-side-nav-surface);
+  border-color: transparent;
```

Comment:
since we're not setting anywhere the border width, I imagine this is needed because it's a button in some cases, right? in that case, probably better to make it clear/explicit adding a comment